packages/mui-base/src/Popover/Root/PopoverRoot.types.ts

```diff
@@ -0,0 +1,82 @@
+import type { FloatingRootContext, OpenChangeReason } from '@floating-ui/react';
+import type { TransitionStatus } from '../../utils/useTransitionStatus';
+import type { GenericHTMLProps } from '../../utils/types';
+
+export interface PopoverRootContextValue {
+  open: boolean;
+  openOnHover: boolean;
+  setOpen: (open: boolean, event?: Event, reason?: OpenChangeReason) => void;
+  triggerElement: Element | null;
+  setTriggerElement: (el: Element | null) => void;
+  positionerElement: HTMLElement | null;
+  setPositionerElement: (el: HTMLElement | null) => void;
+  delay: number;
+  closeDelay: number;
+  delayType: 'rest' | 'hover';
+  instantType: 'dismiss' | 'click' | undefined;
+  mounted: boolean;
+  setMounted: React.Dispatch<React.SetStateAction<boolean>>;
+  transitionStatus: TransitionStatus;
+  titleId: string | undefined;
+  setTitleId: React.Dispatch<React.SetStateAction<string | undefined>>;
+  descriptionId: string | undefined;
+  setDescriptionId: React.Dispatch<React.SetStateAction<string | undefined>>;
+  rootContext: FloatingRootContext;
```

Comment:
Let's call it `floatingRootContext` as having `rootContext` in `PopoverRootContext` looks weird

```suggestion
  floatingRootContext: FloatingRootContext;
```